.github/workflows/callable-e2e-tests.yml

```diff
@@ -64,10 +72,11 @@ jobs:
           GH_PAT: ${{ secrets.GH_PAT }}
       - name: Load Verdaccio with AmplifyJs
         uses: ./aws-amplify/.github/actions/load-verdaccio-with-amplify-js
-      - name: Run E2E test
+      - name: Run cypress tests for ${{ inputs.test_name }}
+
         run: |
           ../aws-amplify/.circleci/retry-yarn-script.sh -s \
-          'ci:test ${{ inputs.framework }} ${{ inputs.category }} ${{ inputs.sample_name }} ${{ inputs.spec }} ${{ matrix.browser }}'
+          "ci:test ${{ inputs.framework }} ${{ inputs.category }} ${{ inputs.sample_name }} ${{ inputs.spec }} ${{ matrix.browser }} dev ${{ inputs.amplifyjs_dir == true && env.AMPLIFY_DIR || ''}}"
```

Comment:
if `amplifyjs_dir: true` is provided add AMPLIFY_DIR when calling the integ test else don't